frontend/src/components/videoPlayer/AddComment.js

```diff
@@ -20,27 +24,99 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-const AddComment = () => {
+const AddComment = ({ data }) => {
   const classes = useStyles();
+  const [comment, setComment] = useState('');
+  const [alert, setAlert] = useState({
+    show: false,
+    message: '',
+    severity: '',
+  });
+
+  const onChangeComment = (e) => {
+    const comment = e.target.value;
+    setComment(comment);
+  };
+
+  const handleComment = async (e) => {
+    e.preventDefault();
+    try {
+      await movieService.createComment(comment, data.movie.imdbCode);
+      setAlert({
+        show: true,
+        message: 'Thanks for the comment!',
+        severity: 'success',
+      });
+    } catch (err) {
+      switch (err.response.data.statusCode) {
+        case 400:
+          setAlert({
+            show: true,
+            message: 'Comment format is wrong',
```

Comment:
remember translation, and also add what right format is, e.g. between 1 - 250 chars